tests/test_circ_ref.py

```diff
@@ -39,10 +32,4 @@ def test_srqk_with_uccsdpqe_ref(self):
         alg2.run(dt=1.0, s=12)
 
         Egs = alg2.get_gs_energy()
-        self.assertLess(abs(Egs-Efci), 5.0e-6)
-
-
-
-
-if __name__ == '__main__':
-    unittest.main()
+        assert Egs == approx(Efci, abs=5.0e-7)
```

Comment:
```suggestion
        assert Egs == approx(Efci, abs=5.0e-6)
```

Merge after you change this back.